api/src/services/tickets/tickets.js

```diff
@@ -19,16 +15,15 @@ export const ticket = async ({ id }) => {
   let result = await db.ticket.findUnique({
     where: { id },
     include: {
-      TicketNote: true,
-    },
+      User: true
+    }
```

Comment:
This is how we load in references that will be available.  More here https://www.prisma.io/docs/concepts/components/prisma-client/relation-queries#nested-reads